src/Actions/MakeQueueTenantAwareAction.php

```diff
@@ -79,4 +84,18 @@ protected function findTenant(JobProcessing $event): Tenant
 
         return $tenant;
     }
+
+    protected function getJobFromQueueable($queueable)
```

Comment:
Could you add an `object` typehint and `string` return type.

Reply:
Added type hint for object. Sorry i'm not used to using type hints a lot for primitives. Either way, there is no union type for the return type as it can return both a string (FQCN) as well as an object. Hence the reason I used class reflection to check whether or not they implement the necessary interfaces as it can accept a string or object.